graderx/graders/manager.py

```diff
@@ -168,22 +186,81 @@ def get_course_data(course_id):
         course_data = get_courses_config()[course_id]
     except KeyError:
         raise CourseNotFoundError()
+    course_data['name'] = course_id
+    return course_data
+
+
+def get_course_data_with_test_cases(course_id):
+    course_data = get_course_data(course_id)
     for index, lab in enumerate(course_data['labs']):
         course_data['labs'][index]["test_cases"] = stdout_common.get_test_cases(course_id, lab["name"])
```

Comment:
using an `enum` for the keys would be pretty helpful, as typos in those keys will take so long to debug